slurm/1_Advanced/GPUs/README.md

```diff
@@ -1,8 +1,40 @@
 # Example script for GPU usage
 
- TODO: write me
+This directory includes a [basic example](./BasicExample.sh) that shows how to request and run a GPU-enabled job on CCR's clusters.  
 
 ## How to use
 
-TODO: write me
+CCR's academic cluster has a mix of compute nodes from various generations of hardware with a variety of GPU types in them. Most of these compute nodes have either 1 or 2 GPUs, while one node has 12 
+A16 GPUs.
+
+The provided [script](./BasicExample.sh) is a minimal Slurm example that uses a GPU. Make sure to modify parts of the script to suit your GPU requirements and specific needs. For 
```

Comment:
```suggestion
The provided [script](./BasicExample.sh) is a minimal Slurm example that requests a GPU. Make sure to modify parts of the script to suit your GPU requirements and specific needs. For 
```

Reply:
The main reason for this wording change is a user needs to request the GPU in order to run on it but not all code runs correctly on the GPU  :)